site/hack/release-docs.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# How to run the command to generate new documentation of impkg from develop with the version v0.5.0
+# ./hack/release-docs.sh imgpkg v0.5.0
+
+TOOL=$1
+NEW_VERSION=$2
+FROM_VERSION=${3:-develop}
+
+CONTENT_DIRECTORY=content/${TOOL}/docs
+TOC_DIRECTORY=data/${TOOL}/docs
+
+# don't run if there's already a directory for the target docs version
+if [[ -d $CONTENT_DIRECTORY/$NEW_VERSION ]]; then
+    echo "ERROR: $CONTENT_DIRECTORY/$NEW_VERSION already exists"
+    exit 1
+fi
+
+LATEST_VERSION=$(yq e ".params.${TOOL}.version_latest" config.yaml)
+# don't run if cannot find the origin version
+if [[ ! -d $CONTENT_DIRECTORY/$FROM_VERSION ]]; then
+    echo "ERROR: $CONTENT_DIRECTORY/$FROM_VERSION from version folder does not exist"
+    exit 1
+fi
+
+PREVIOUS_VERSION="${LATEST_VERSION}"
+COPY_FROM_FOLDER="${CONTENT_DIRECTORY}/${LATEST_VERSION}/"
+if [ "${FROM_VERSION}" != "develop" ] && [ "${FROM_VERSION}" != "${LATEST_VERSION}" ]; then
+  PREVIOUS_VERSION=$(head -n8 "${CONTENT_DIRECTORY}"/${FROM_VERSION}/_index.md | yq e '.cascade.version' -)
```

Comment:
Same `yq` dependency as earlier:

```console
$ hack/release-docs.sh ytt v0.38.1 v0.38.0
usage: yq [-h] [--yaml-output] [--yaml-roundtrip] [--width WIDTH]
          [--indentless-lists] [--in-place] [--version]
          [jq_filter] [files ...]
yq: error: argument files: can't open '.cascade.version': [Errno 2] No such file or directory: '.cascade.version'
```

```suggestion
  PREVIOUS_VERSION=$(ytt -f version.yml=<(echo -e "#@ load('@ytt:data', 'data')\n--- #@ data.values.cascade.version") --data-values-file <(head -n8 content/ytt/docs/v0.38.0/_index.md))
```

Reply:
done